src/main/java/stelnet/storage/view/StorageTabViewFactory.java

```diff
@@ -89,7 +89,7 @@ private void addEmptyData(List<AbstractRenderable> elements, List<StorageData> s
     }
 
     private void addStorageData(List<AbstractRenderable> elements, List<StorageData> storageData) {
-        if (!hasStorage()) {
+        if (!hasStorage() || storageData.isEmpty()) {
```

Comment:
What confuses me is this check should not be needed for iterating over Collection. If it is empty, then for each will not run.

Reply:
Found the bug. It's actually the code to remove last spacer element, which does not exist if it is empty.